components/dashboard/src/admin/License.tsx

```diff
@@ -202,7 +203,8 @@ function communityPlan(userCount: number, seats: number, fallbackAllowed: boolea
         }
     };
 
-    const aboveLimit: boolean = userCount > seats;
+    // seats === 0 means unlimited number of users
+    const aboveLimit: boolean = seats === 0 ? false : userCount > seats;
```

Comment:
I wonder if this could be de-duplicated as a single, `isAboveLimit(userCount, seats)` function. 💭 But not very important.